### PR TITLE
calendar

### PR DESCRIPTION
**Pull request by BOHDANLIUYUI**
- [DEMO LINK](https://BOHDANLIUYUI.github.io/layout_calendar/)
- [TEST REPORT LINK](https://BOHDANLIUYUI.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

**Review by etojeDenys — CHANGES_REQUESTED** [review]
it looks incredible
but I should have the opportunity to change the number of days just by changing the modifier
image

File: src/styles/blocks/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
+.calendar {
+  box-sizing: content-box;
```
Comment (etojeDenys) [inline src/styles/blocks/calendar.scss:2]: move to * selector

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/styles/blocks/calendar.scss (hunk 1 above)
Comment (BOHDANLIUYUI) [reply src/styles/blocks/calendar.scss:2]: When removed "box-sizing" in  calendar block it crush 
image

Comment (sTorba24) [reply src/styles/blocks/calendar.scss:2]: Mentor meant that you need to write this style to * selector 
like
```
* {
  box-sizing: border-box;
}
```

**Review by sTorba24 — APPROVED** [review]
Good work!

File: src/styles/blocks/calendar.scss (hunk 1 above)
Comment (BOHDANLIUYUI) [reply src/styles/blocks/calendar.scss:2]: Yes, when I  write this style to * selector, it no work

**Pull request closed without merging**